fmriprep/__about__.py

```diff
@@ -89,14 +89,16 @@
     'nilearn!=0.5.0,!=0.5.1',
     'nipype>=1.1.6',
     'nitime',
-    'niworkflows @ git+https://github.com/effigies/niworkflows.git@5e9daf0b08fcad2b17350a71ecaf71fa9ce4eff7',
+    'niworkflows @ '
+    'git+https://github.com/effigies/niworkflows.git@5e9daf0b08fcad2b17350a71ecaf71fa9ce4eff7',
     'numpy',
     'pandas',
     'psutil>=5.4',
     'pybids<0.8.0a0,>=0.7.1',
     'pyyaml',
     'scikit-image',
-    'smriprep @ git+https://github.com/poldracklab/smriprep.git@f1cfc37bcdc346549dbf1d037cdade3a3b32d5de',
+    'smriprep @ ',
```

Comment:
this was potentially working. please note that comma breaking up the string concatenation.